source/_integrations/zwave_js.markdown

```diff
@@ -190,7 +190,7 @@ Do this if you have an existing Z-Wave network and want to use a new adapter. Th
 4. Select **Migrate to a new adapter**.
 
    - To confirm device reset, select **Submit**.
```

Comment:
We won't reset the adapter.

Reply:
oh, duh. Thank you for the catch. :pray: